### ISSUE TITLE
Pagefind UI: mark matches in metadata items

### ISSUE THREAD

**Issue by holamgadol** [CloudCannon/pagefind#160]
Pagefind is a great tool, thanks for that. I wish It will be developing in the future.

Is it possible to mark matches not only in _excerpt_ field, but also in metadata items, like titles and tags? It will gain greatly the visual representation of search results.

**Comment by bglw** [comment]
Yes! Thanks for pointing that out — I'll need to find a nice way to provide that data through the JS API, but that is very doable 🙂
